client/core/core.go

```diff
@@ -4658,6 +4658,24 @@ func (c *Core) connectWallets() {
 					db.ErrorLevel, wallet.state()))
 				return
 			}
+			if mw, is := wallet.Wallet.(asset.FundsMixer); is {
+				startMixing := func() error {
+					stats, err := mw.FundsMixingStats()
+					if err != nil {
+						return fmt.Errorf("error checking %s wallet mixing stats: %v", unbip(wallet.AssetID), err)
+					}
+					if err := c.connectAndUnlock(crypter, wallet); err != nil {
+						return fmt.Errorf("error unlocking %s wallet for mixing: %v", unbip(wallet.AssetID), err)
+					}
```

Comment:
Don't need to unlock if `!stats.Enabled`

Reply:
Even disabled can move funds if any are still left in mixed and unmixed accounts. The stats could include this info however...